news/13440-add-yn-prompt

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Add a Y/N prompt warning users that conda env remove  -p <prefix_path> and conda remove --all -p <prefix_path> deletes not only the conda environment at <prefix_path> but all the other contents if present.
```

Comment:
```suggestion
* Add a Y/N prompt warning users that `conda env remove` and `conda remove --all` deletes not only the conda packages but all other files present in the specified environment. (#12222 via #13440)
```

Reply:
Hey, @kenodegard !
What we are trying to tell the users in that the entire directory that contains the environment will be deleted. I am thinking of the case when a user creates an environment in a non empty directory and then tries to delete the environment. In this case they should know that the directory in which the environment lives will also be deleted.

Reply:
ok so we modify it to:

```suggestion
* Add a Y/N prompt warning users that `conda env remove` and `conda remove --all` deletes not only the conda packages but all other contents in the specified environment. (#12222 via #13440)
```